boards/metroesp32-s2.json

```diff
@@ -0,0 +1,33 @@
+{
+  "build": {
+    "arduino":{
+      "ldscript": "esp32s2_out.ld"
+    },
+    "core": "esp32",
+    "f_cpu": "240000000L",
+    "f_flash": "80000000L",
+    "flash_mode": "dio",
+    "mcu": "esp32s2",
+    "variant": "esp32s2"
```

Comment:
Given...  [adafruit_feather_esp32s2](https://github.com/espressif/arduino-esp32/tree/master/variants/adafruit_feather_esp32s2)  should this be... "variant" : "adafruit_feather_esp32s2"  so correct pins_Arduino.h and other variant content gets used during build?

Reply:
> @valeros @makermelissa Given... [espressif/arduino-esp32/.../variants/adafruit_metro_esp32s2](https://github.com/espressif/arduino-esp32/tree/master/variants/adafruit_metro_esp32s2) should this be... "variant" : "adafruit_metro_esp32s2" so correct pins_Arduino.h and other variant content gets used during build? If so, does adafruit_feather_esp32s2 and other boards need variant values updated?

Asking because am trying to add Adafruit QT Py ESP32-S2 board in https://github.com/platformio/platform-espressif32/pull/720 and had to set variant key with value of adafruit_qtpy_esp32s2 in order for PlatformIO to compile the correct pins_Ardunio.h.

Reply:
yes we have like half a dozen esp32-s2 boards now - all of them need unique variant files! :)